agent/resources/resources_windows.go

```diff
@@ -0,0 +1,52 @@
+// +build windows
```

Comment:
In line with my other comment about building on mac, I have a few suggestions:
* rename this to `resources_unsupported'
* build tag `!linux`
* return nil instead of returning errors

Windows and Unix are unsupported. We want agent to compile on mac (not necessarily for running agent, but testing and compiling is important to folks using IDEs on mac). Hence the change to build tag / filename.